src/runtime/ui-particle.ts

```diff
@@ -11,24 +11,35 @@
 import {XenStateMixin} from '../../modalities/dom/components/xen/xen-state.js';
 import {UiParticleBase} from './ui-particle-base.js';
 import {Handle} from './handle.js';
-import {Runnable} from './hot.js';
+import {Runnable, Dictionary} from './hot.js';
```

Comment:
Just checking that you intended to include the changes in this file in this PR.

Reply:
I did, but it was unwise. Will move to separate PR.